src/style.scss

```diff
@@ -297,6 +297,30 @@ $maxCaptionHeight: 150px;
   right: 0;
   bottom: 0;
   left: 0;
+  .imagePrev & {
+    display: none;
+  }
+  .imageNext & {
+    display: none;
+  }
+}
+
+.errorContainer {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #ff0000;
```

Comment:
I think `#fff` would be a good color in this context. Bright red is a bit too alarming, and I think the fact that text comes up instead of an image is warning enough that something went wrong.